app/main.py

```diff
@@ -1,6 +1,36 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
+CITY_FILTERING = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    url = BASE_URL + f"?key={API_KEY}" + f"&q={CITY_FILTERING}"
+    res = requests.get(url)
```

Comment:
You can pass `key` and `q` as params to `request.get`
https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls